test/integration/targets/nxos_vxlan_vtep/tests/common/sanity.yaml

```diff
@@ -23,6 +48,10 @@
         host_reachability: true
         source_interface: Loopback0
         source_interface_hold_down_time: 30
+        global_ingress_replication_bgp: "{{global_ingress_replication_bgp|default(omit)}}"
+        global_suppress_arp: "{{global_suppress_arp|default(omit)}}"
+        global_mcast_group_L3: "{{global_mcast_group_L3|default(omit)}}"
+        #global_mcast_group_L2: "{{global_mcast_group_L2|default(omit)}}"
```

Comment:
Did you mean to leave this commented out?

Reply:
Yes. When global_ingress_replication_bgp is present, global_mcast_group_L2 cannot be used. These are mutually exclusive. I tested it on its own but this cannot be uncommented in this task.
See https://github.com/saichint/ansible/blob/vxlan_vtep/lib/ansible/modules/network/nxos/nxos_vxlan_vtep.py#L339

Reply:
We should have a separate test for each if possible.  Otherwise this will never get tested.

Reply:
Added new tasks for testing global_mcast_group_L2